.pylintrc

```diff
@@ -373,7 +373,7 @@ max-locals=15
 max-returns=10
 
 # Maximum number of branch for function / method body
-max-branches=12
+max-branches=15
```

Comment:
Allow more `if` branches instead of suppressing the `too-many-branches` error inline.

cc @villebro @john-bodley @etr2460

Reply:
Yikes! This is an anti-pattern in my opinion. The check is there for a reason to prevent non-readable code. If one must override this, it should be done locally and not globally.